aws-codeguruprofiler-profilinggroup/src/test/java/software/amazon/codeguruprofiler/profilinggroup/CreateHandlerTest.java

```diff
@@ -43,83 +53,226 @@
     @Mock
     private Logger logger;
 
+    private CreateHandler subject;
+
     private ResourceHandlerRequest<ResourceModel> request;
 
+    private final String profilingGroupName = "Silver-2020";
+    private final String clientToken = "clientTokenXXX";
+    private final CreateProfilingGroupRequest createPgRequest = CreateProfilingGroupRequest.builder()
+            .profilingGroupName(profilingGroupName)
+            .clientToken(clientToken)
+            .build();
+
     @BeforeEach
     public void setup() {
         proxy = mock(AmazonWebServicesClientProxy.class);
         logger = mock(Logger.class);
-
-        request = makeValidRequest();
+        subject = new CreateHandler();
```

Comment:
Minor: All of these can be inlined with the filed declaration itself.

Reply:
Good point, thanks, I will update.